Classes/git/PBGitRevList.mm

```diff
@@ -200,13 +200,13 @@ - (void) setupEnumerator:(GTEnumerator*)enumerator
 		}
 	}
 
-//	NSArray *sortedBranchesAndTags = [[enumCommits allObjects] sortedArrayWithOptions:NSSortStable usingComparator:^NSComparisonResult(id obj1, id obj2) {
-//		GTCommit *branchCommit1 = obj1;
-//		GTCommit *branchCommit2 = obj2;
-//
-//		return [branchCommit2.commitDate compare:branchCommit1.commitDate];
-//	}];
-	NSArray *sortedBranchesAndTags = [enumCommits allObjects];
+	NSArray *sortedBranchesAndTags = [[enumCommits allObjects] sortedArrayWithOptions:NSSortStable usingComparator:^NSComparisonResult(id obj1, id obj2) {
+		GTCommit *branchCommit1 = obj1;
+		GTCommit *branchCommit2 = obj2;
+
+		return [branchCommit1.commitDate compare:branchCommit2.commitDate];
```

Comment:
I have to admit I do not understand the surrounding code and what sort it tries to achieve … but this seems to revert the sort order.

WIth the preceding commits and reverts changing this behaviour, I am a bit confused, so perhaps an explanation would be good.

(As it’s unclear to me from which direction the resulting list will be traversed, I cannot figure out whether newest first or oldest first would be the correct way to go here.)